src/quoteFeedXignite.js

```diff
@@ -2151,8 +2153,8 @@ CIQ.QuoteFeed.Xignite.Templates = {
 	HistoricalRates: {
 		host: {
 			protocol: "https://",
-			server: "www.xignite.com",
-			path: "/www_xignite"
+			server: "rates.xignite.com",
+			path: "/rates_xignite"
```

Comment:
So I'm not seeing how you get "/rates_xignite" from the Xignite speadsheet and your table in the comments. Can you post the Xignite api page or whatever you used to get this path?

Reply:
<div dir="ltr" class="gmail_attr" style="caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-family: Helvetica; font-size: 14px; font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;">On Sun, Sep 11, 2022 at 7:26 PM Rob Bruckheimer &lt;<a href="mailto:rob@cosaic.io">rob@cosaic.io</a>&gt; wrote:<br></div><blockquote class="gmail_quote" style="caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-family: Helvetica; font-size: 14px; font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none; margin: 0px 0px 0px 0.8ex; border-left-width: 1px; border-left-style: solid; border-left-color: rgb(204, 204, 204); padding-left: 1ex;"><div style="overflow-wrap: break-word;">Hi Daniel,<div><br></div><div>Thanks for the detail.</div><div>Upon further review I have found some additional entries, one of which corresponds to what you saw for InvestEdge:</div><div><br></div><div><b><font size="3">API<span style="white-space: pre-wrap;">							</span>Server<span style="white-space: pre-wrap;">						</span>Method</font></b></div><div>

xFutures | www | GetDelayedFuture
-- | -- | --
xFutures | www | GetHistoricalCommodityRange
xFutures | www | GetHistoricalFutureRange
xGlobalQuotes/v3 | chartiq | GetChartBars
xBATSRealTime | chartiq | GetChartBars
xglobalindices | chartiq | GetChartBars
xglobalindicesrealtime | chartiq | GetChartBars

<div><br></div></div></div></div></div></div></div><br></div></div></blockquote></div><span id="m_-5893036557521539200m_7958884842462457779m_5765391622182044043m_7281535183725451544cid:f_l7f8au9d0">&lt;Xignite_API_calls.xlsx&gt;</span></div></blockquote></div></div></div></blockquote></div></div></blockquote></div></div></div></blockquote>On Sun, Sep 11, 2022 at 7:26 PM Rob Bruckheimer <[rob@cosaic.io](mailto:rob@cosaic.io)> wrote:
Hi Daniel,

Thanks for the detail.
Upon further review I have found some additional entries, one of which corresponds to what you saw for InvestEdge:

API							Server						Method
xFutures	www	GetDelayedFuture
xFutures	www	GetHistoricalCommodityRange
xFutures	www	GetHistoricalFutureRange
xGlobalQuotes/v3	chartiq	GetChartBars
xBATSRealTime	chartiq	GetChartBars
xglobalindices	chartiq	GetChartBars
xglobalindicesrealtime	chartiq	GetChartBars


Based on the above, the spreadsheet previously provided, and your feedback, I have collated the changes required as follows:

API	Current Server	New server	Method
xBATSRealTime	chartiq	batsrealtime	GetChartBars
xEarningsCalendar	www	earningscalendar	GetEarningCalendarItemsAsString
xEdgar	www	edgar	SearchFilings
xGlobalCurrencies	chartiq	globalcurrencies	ConvertRealTimeValue
xGlobalCurrencies	chartiq	globalcurrencies	GetChartBars
xGlobalCurrencies	chartiq	globalcurrencies	GetRealTimeRates
xglobalindices	chartiq	globalindices	GetChartBars
xglobalindicesrealtime	chartiq	globalindicesrealtime	GetChartBars
xGlobalMetals	chartiq	globalmetals	GetChartBars
xGlobalMetals	chartiq	globalmetals	GetRealTimeMetalQuote
xGlobalQuotes/v3	chartiq	globalquotes	GetChartBars
xRates	www	rates	GetHistoricalRateFamily
xRates	www	rates	GetLatestRateFamily

Please let me know if this looks right to you, as well as the date by which this work must be completed.  It’s quite possible we will need to send updated ChartIQ libraries to US News and Yodlee for these changes.

Thanks,

Rob Bruckheimer
Chief Software Architect

Cosaic

+1 800.821.8147
[rob@cosaic.io](mailto:rob@cosaic.io)








On Sep 9, 2022, at 12:01 PM, Daniel Yee <[dyee@xignite.com](mailto:dyee@xignite.com)> wrote:

Hi Rob,

Thank you for your patience. The mutual client is Invest Edge. After an internal discussion with our team, we believe Chart IQ could drop the '-chartiq' part of the server name and use the regular domain names. However this change is not urgent. 

Some of the legacy services (xFutures, xCalendar, and xInsider) have been removed and no longer available.  
For xEarningsCalendar and xRates, it's better to use [earningscalendar.xignite.com](http://earningscalendar.xignite.com/) and [rates.xignite.com](http://rates.xignite.com/) instead of www.

Please let us know if you have any questions regarding this. 

Best,
Daniel 

On Mon, Aug 29, 2022 at 1:56 PM Rob Bruckheimer <[rob@cosaic.io](mailto:rob@cosaic.io)> wrote:
Hi Daniel,

Are we ok with all these server names, or do we need to make changes to any of them?
Do we need to change the casing of the API names, or will they continue to work as is?
We are all set then on the TLS.  We made the change in the Java app to 1.2 some time ago.
As fo the log of the ChartIQ server being requested for xGlobalQuotes, can you specify the customer, so we know where to look for this?

Thanks,

Rob Bruckheimer
Chief Software Architect

Cosaic

+1 800.821.8147
[rob@cosaic.io](mailto:rob@cosaic.io)








On Aug 29, 2022, at 4:49 PM, Daniel Yee <[dyee@xignite.com](mailto:dyee@xignite.com)> wrote:

Hi Rob,

Thank you for sending us this information. I updated some of the API names in column A (to upper casing). 

After our October 31, 2022 deployment, we will only support TLS 1.2 and above. 

We found this URL ([chartiq.xignite.com/v3/xGlobalQuotes.csv/GetChartBars](http://chartiq.xignite.com/v3/xGlobalQuotes.csv/GetChartBars)) in one of our client's usage data from June 2022. However, we have not seen this in last week's recent usage. Can you please confirm if this is a valid URL on your end or should they be using "GlobalQuotes-chartiq" instead? 

Best,
Daniel

On Thu, Aug 25, 2022 at 12:30 PM Rob Bruckheimer <[rob@cosaic.io](mailto:rob@cosaic.io)> wrote:
Hello,
Thanks for the discussion last Friday.
Please find attached a list of API names, server names, and methods we are using across our codebase to communicate with your APIs.
We understand some of these APIs may no longer be supported; we didn’t check to see what was in use, only what was programmed.
Can you please review and determine if any of the server names need to change, or if we need to make sure that the API names are capitalized properly?
Also, I wanted to confirm about the TLS upgrade.  What versions of TLS will be supported once the upgrade in October happens?  I thought we discussed 1.3, but within our notes made prior to our call it says 1.2.  Just wanted to clarify.
Thanks,
Rob

Rob Bruckheimer
Chief Software Architect

Cosaic

+1 800.821.8147
[rob@cosaic.io](mailto:rob@cosaic.io)


<Xignite_API_calls.xlsx>

Reply:
See above email chain, specifically:
For xEarningsCalendar and xRates, it's better to use [earningscalendar.xignite.com](http://earningscalendar.xignite.com/) and [rates.xignite.com](http://rates.xignite.com/) instead of www.